targets/simple_switch/tests/pcap_mock/Makefile.am

```diff
@@ -0,0 +1,10 @@
+AM_CXXFLAGS = -g -pthread -std=c++20
```

Comment:
The codebase has not been updated to C++20 yet, so please stick to C++17, or submit a separate change to update the code base to C++20